core/modules/widgets/draggable.js

```diff
@@ -114,10 +126,19 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 DraggableWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.tag || changedAttributes.selector || changedAttributes.dragimagetype || changedAttributes.enable || changedAttributes.startactions || changedAttributes.endactions) {
+	if(changedAttributes.tag || changedAttributes.selector || changedAttributes.dragimagetype || changedAttributes.startactions || changedAttributes.endactions) {
 		this.refreshSelf();
 		return true;
 	} else {
+		if(changedAttributes.enable) {
+			this.dragEnable = this.getAttribute("enable","yes") === "yes";
+			this.makeDraggable(this.domNodes[0]);
+			if(!this.dragHandleSelector && this.dragEnable && !this.domNodes[0].classList.contains("tc-draggable")) {
+				this.domNodes[0].classList.add("tc-draggable");
```

Comment:
We have a utility function for toggling classes that would be useful to use here. Also, it might be more robust for the widget to track its own internal state (enabled/disabled) rather than check the classes of the DOM node to ascertain this.